noid_ark.py

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Do you intend to provide an example to use this library? Please create a examples and put this file with instruction in it. Thanks for the PR.

Reply:
Apologies, I just meant to merge this into my fork not master. I'll delete this pull request.